src/utils/fileLoaders.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const path = require('path');
+const jsyaml = require('js-yaml');
+
+const cwd = process.cwd();
+
+const file = (refValue, options) => {
+    let refPath = refValue;
```

Comment:
is this needed? why not just do immutable assignment in a ternary statement below?

Reply:
changed code a bit